### PR TITLE
feat(operator): add events to the jivavolume controller

### PR DESCRIPTION
**Pull request by shubham14bajpai**
Signed-off-by: shubham <shubham.bajpai@mayadata.io>

This PR adds to the jiva volume controller. This will enable a single point of debugging and help users get usuable information without going through long log files. This PR also improves the logging for env and errors.
Example events:
```
Events:
  Type     Reason          Age                  From                   Message
  ----     ------          ----                 ----                   -------
  Warning  Bootstrap       34m (x25 over 35m)   jivavolume-controller  failed to bootstrap volume, due to error: failed to get volume policy example-jivavolumepolicy: JivaVolumePolicy.openebs.io "example-jivavolumepolicy" not found
  Warning  ReplicaScaleup  2s (x6 over 12s)     jivavolume-controller  failed to scaleup volume, only single replica scaleup is allowed, desired: 3 actual: 1
```

### CODE REVIEW

File: cmd/manager/main.go [hunk 1]
```diff
@@ -20,6 +20,7 @@ import (
 	"flag"
 	"fmt"
 	"os"
+	env "runtime"
```
Comment (prateekpandey14) [inline cmd/manager/main.go:23]: why env ?

Comment (shubham14bajpai) [reply cmd/manager/main.go:23]: runtime was already used by another import. And this helps us log the OS/ARCH & Go version and path which are env on which the image was built.

Comment (shubham14bajpai) [reply cmd/manager/main.go:23]:
```
time="2021-03-11T10:03:46Z" level=info msg="Go Version: go1.15.5"
time="2021-03-11T10:03:46Z" level=info msg="Go OS/Arch: linux/amd64"
time="2021-03-11T10:03:46Z" level=info msg="Version of jiva-operator: scaleup-dev"
```

Comment (prateekpandey14) [reply cmd/manager/main.go:23]: ok if runtime alias was used then its fine.

**Review by prateekpandey14 — APPROVED** [review]
lgtm

**Pull request merged**